### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by knightcube**
First steps for issue #329 

**Changes**: Added Screenshots of the app and updated the dependencies list in README.md

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [app-debug.zip](https://github.com/fossasia/neurolab-android/files/3332794/app-debug.zip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +79,22 @@ If you are a tester and want to test the app, you have two ways to do that:
...
+* *[JTransforms](https://mvnrepository.com/artifact/net.sourceforge.jtransforms/jtransforms/2.4.0)*
+* *[Java Open GL](https://mvnrepository.com/artifact/org.jogamp.jogl/jogl-all-main/2.3.2)*
+* *[Circle ImageView](https://github.com/hdodenhof/CircleImageView)*
+* *[MP Android Chart - Chart Library](https://github.com/PhilJay/MPAndroidChart)*
+* *[USB Serial Controller](https://github.com/felHR85/UsbSerial)*
+* *[App Intro](https://github.com/AppIntro/AppIntro)*
+* *[JFreeChart](https://mvnrepository.com/artifact/org.jfree/jfreechart/1.0.14)*
+* *[OpenCSV](https://mvnrepository.com/artifact/com.opencsv/opencsv/4.6)*
+* *[Gson](https://mvnrepository.com/artifact/com.google.code.gson/gson/2.8.5)*
+
```
Comment (jddeep) [inline README.md]: @knightcube Please remove these extra lines below. One extra line is enough.

**Review by jddeep — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
Also keep this PR directed to #329 . Make another PR directing #328 and #326 together if you like.

File: README.md (hunk 1 above)
Comment (jddeep) [reply README.md]: Also address Codacy issues. They seem resolvable.

**Review by jddeep — COMMENTED** [review]
Once done with Codacy issues and also updated the whole Readme file, update PR body with "Fixes#Issue" and squash all commits into one commit.

File: README.md [hunk 2]
```diff
@@ -19,8 +19,25 @@ A difference to existing projects like OpenBCI is that it will not be necessary
 ## Screenshots
 <table>
         <tr>
-<td><img src = "https://user-images.githubusercontent.com/20669217/55556050-432cb580-5704-11e9-85dd-d223c16d7a38.png" height = "500" width="250"></td>
+
+  <td><img src = "https://user-images.githubusercontent.com/22871961/60230665-5343d380-98b4-11e9-8bc1-9f39756ad964.png" height = "500" width="250"></td>
```
Comment (CloudyPadmal) [inline README.md]: Let's keep images in docs/images folder and link them here.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Rather than "Screenshot_1231313123123123.png" let's name them properly. "Startscreen1.png" etc.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
What's screen5? what's screen6? Please have a look at how images are labeled in PSLab android app. https://github.com/fossasia/pslab-android/tree/development/docs/images

Still the readme links are not from doc/images

Also squash commits into one with a valid name for the PR

File: README.md [hunk 3]
```diff
@@ -18,9 +18,25 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+  <td><img src = "/docs/images/start_screen_1.png" height = "500" width="250"></td>
+
+  <td><img src = "/docs/images/start_screen_2.png" height = "500" width="250"></td>  
+
+  <td><img src = "/docs/images/start_screen_3.png" height = "500" width="250"></td>
+  </tr>
+
+  <tr>
+  <td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
+  <td><img src = "/docs/images/nav_menu.png" height = "500" width="250"></td>
```
Comment (CloudyPadmal) [inline README.md:33]: Add one more image here. Otherwise it looks like an empty cell when viewed.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (knightcube) [reply README.md:33]: Sure. Should I add the spectrum activity screenshot?

Comment (knightcube) [reply README.md:33]: Demo
How about this as demo.gif? I got this from one of the PRs

Comment (jddeep) [reply README.md:33]: Yeah you can add it.

**Comment by knightcube** [comment]
I have made the required changes. Let me know if there are any more problems in this PR. Will be happy to fix them. Kindly review @CloudyPadmal

**Review by CloudyPadmal — APPROVED** [review]

**Review by jddeep — APPROVED** [review]

**Pull request merged**